crates/rbuilder/src/live_builder/order_input/mod.rs

```diff
@@ -334,33 +334,41 @@ where
     let handle = tokio::spawn(async move {
```

Comment:
Usually this jobs need to be spawned as blocking and not as async tasks and its time consuming and blocks executor.

Reply:
Its probably fine here as this job happens between slots.